src/model/formats/foreflight-writer.ts

```diff
@@ -100,8 +100,9 @@ export class ForeFlightWriter {
           case ChecklistItem_Type.ITEM_CAUTION:
           case ChecklistItem_Type.ITEM_WARNING: {
             const text = ForeFlightUtils.getChecklistItemPrefix(itemEFIS.type) + itemEFIS.prompt;
-            const [lastItemFF, lastItemEFIS] = accumulator[accumulator.length - 1] || [];
-            if (accumulator.length && lastItemEFIS.indent < itemEFIS.indent) {
+
+            const [lastItemFF, lastItemEFIS] = accumulator.at(-2) ?? [];
```

Comment:
Do you really mean to get the second-to-last item here? That's different from the previous logic (and doesn't match the variable names). (or maybe it's just that this change is really part of b9c95ae614703e5a279c525176ffe5f62eef622b ?)

Reply:
Oh, sorry, this is what you get for faking commit history from a very dirty working copy... I updated the commits to make sense now.